src/gettingstarted/own-code/routes-configuration.md

```diff
@@ -0,0 +1,57 @@
+
+```.
+├── .platform
+│   ├── routes.yaml
+│   └── services.yaml
+├── .platform.app.yaml
+└── < application code >
+```
+
+A minimal configuration `.platform/routes.yaml` for all languages will look very similar:
+
+{% codetabs name="PHP", type="php", url="https://raw.githubusercontent.com/platformsh/template-php/master/.platform/routes.yaml" -%}
+
+{%- language name="Go", type="js", url="https://raw.githubusercontent.com/platformsh/template-golang/master/.platform/routes.yaml" -%}
+
+{%- language name="Python", type="py", url="https://raw.githubusercontent.com/platformsh/template-python3/master/.platform/routes.yaml" -%}
+
+{% language name="Node.js", type="js", url="https://raw.githubusercontent.com/platformsh/platformsh-example-nodejs/master/.platform/routes.yaml" -%}
+
+{%- endcodetabs %}
+
+Configuring the routes can be done using either an absolute URL or a URL template as shown in the examples above that have the form `http://www.{default}`, where `{default}` will be substituted by either your configured domain or those automatically generated by Platform.sh.
+
+If you set up a domain of `example.com`, the route configuration `http://www.{default}` will be resolved to `http://www.example.com/`. Your production (`master`) environment's routes will be configured according to these rules, but so will each development environment that you activate.
+
+Each route can then be configured with the following properties:
+
+  * `type`:
+      * `upstream`: serves the application. Takes the form `upstream: <application name>:http`, using the application `name` set in your `.platform.app.yaml`.
+      * `redirect`: configures redirects from `http://{default}` to your application.
+  * `cache`: controls [caching behavior of the route](/configuration/routes/cache.md).
+  * `ssi`: controls whether [Server Side Includes](/configuration/routes/ssi.md) are enabled.
+  * `redirects`: controls [redirect rules](/configuration/routes/redirects.html) associated with the route.
+
+> Each language and framework may have additional attributes that you will need to include in `.platform/routes.yaml` depending on the needs of your application. To find out what else you may need to include to configure your routes, consult
+>
+> * **The [Routes](/configuration/services.md) documentation for Platform.sh**
+>    The documentation goes into far more extensive detail of which attributes can also be included for route configuration, and should be used as your primary reference.   
+> * **Language-specific templates for Platform.sh Projects**
+>    Compare the `.platform/routes.yaml` file from the simple template above to other templates when writing your own.
+
+Now that you have configured your routes, you may be wondering how to actually use the services you configured in the previous step within your application. In the next step, you will be able to see how Platform.sh leverages environment variables to make connecting to services simple.   
```

Comment:
This seems like a non-sequitur segue.  "Now that you've configured routes, you are probably thinking about services!"  Um, no, that's not what was on my mind at all...

Reply:
Going to nix the first sentence.